### PR TITLE
3-AZ audit ES cluster & various fixes

### PR DESCRIPTION
**Pull request by alkar**
Connects to https://github.com/ministryofjustice/cloud-platform/issues/979

### CODE REVIEW

File: terraform/global-resources/elasticsearch.tf [hunk 1]
```diff
@@ -324,4 +337,10 @@ resource "aws_elasticsearch_domain" "test" {
   ]
 }
 TESTPOLICY
+
+  log_publishing_options {
```
Comment (digitalronin) [inline terraform/global-resources/elasticsearch.tf]: Isn't this a duplicate of lines 250-253?

Comment (alkar) [reply terraform/global-resources/elasticsearch.tf]: It's on a different es cluster: `audit_1` vs `test`

File: terraform/global-resources/auth0.tf [hunk 2]
```diff
@@ -5,13 +5,6 @@ resource "auth0_rule" "whitelist-github-orgs" {
   enabled = true
 }
 
-resource "auth0_rule" "whitelist-github-teams" {
-  name    = "whitelist-github-teams"
-  script  = "${file("${path.module}/resources/auth0-rules/whitelist-github-teams.js")}"
-  order   = 20
-  enabled = true
-}
```
Comment (jasonBirchall) [inline terraform/global-resources/auth0.tf]: What was the thinking here?

Comment (jasonBirchall) [reply terraform/global-resources/auth0.tf]: Do we no longer need to whitelist the team, or is this done via the org script now?

Comment (alkar) [reply terraform/global-resources/auth0.tf]: We do not use this rule, I think it was in place when everything was locked down to our team. It's not defined on the tenant.

Comment (jasonBirchall) [reply terraform/global-resources/auth0.tf]: :+1: okay, thanks.

**Pull request merged**